webpack.config.js

```diff
@@ -10,5 +10,8 @@ module.exports = {
             { test: path.join(__dirname, "es6"),
                 loader: "babel-loader" }
         ]
+    },
+    resolve: {
+        modules: ["js", "node_modules"],
```

Comment:
It's possible that you won't need this if you import with the "./" preceding the file name. Give it a try.